Library/.idea/dictionaries/patrickhoette.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="patrickhoette">
```

Comment:
Y you pushing yourself into the project? 😄 @patrickhoette

Reply:
nice catch. Let me look into that.

Reply:
Its safes the dictionaries by users name... why? Its the project dictionary. Well back on the git ignore it goes.